ui/cockpit/src/modules/utils/config.js

```diff
@@ -100,11 +100,16 @@ async function loadPlugins() {
 
 async function loadLocale() {
   const locales = config.locales;
+  const availableLanguages = locales.availableLocales;
   const preferredLanguage = getLanguage();
-  const defaultLanguage = locales[0];
-  const localesToLoad = locales.includes(preferredLanguage)
-    ? [defaultLanguage, preferredLanguage] // Order is important, defaultLanguage always first
-    : [defaultLanguage];
+  const fallbackLanguage = locales.fallbackLocale;
+
+  const isPreferredLanguageAvailable = availableLanguages.includes(
+    preferredLanguage
+  );
```

Comment:
This was broken before, but let's fix it in this ticket as well: when fallback and preffered locale are the same, don't load the same JSON twice.
```suggestion
  const isPreferredLanguageAvailable =
    availableLanguages.includes(preferredLanguage) &&
    preferredLanguage !== fallbackLanguage;
```